source/includes/dont-use-api-key-in-browsers.rst

```diff
@@ -0,0 +1,6 @@
+.. important::
```

Comment:
```suggestion
.. important:: Don't Use API Key Auth from User-Facing Clients
```

think would be beneficial to have smthn like this to underscore the note.

feel free to adjust phrasing.